pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotControllerHealthCheck.java

```diff
@@ -37,8 +38,10 @@
 import org.apache.pinot.controller.BaseControllerStarter;
 import org.apache.pinot.controller.ControllerConf;
 
+import static org.apache.pinot.spi.utils.CommonConstants.SWAGGER_AUTHORIZATION_KEY;
 
-@Api(tags = Constants.HEALTH_TAG)
+
+@Api(tags = Constants.HEALTH_TAG, authorizations = {@Authorization(value = SWAGGER_AUTHORIZATION_KEY)})
 @Path("/")
```

Comment:
should there be additional swagger annotations as everywhere else?

Reply:
Yes good catch. I was thinking maybe health check does not need authz but clearly this is for Swagger so it does not matter